inngest-spring-boot-demo/src/test/java/com/inngest/springbootdemo/CloudModeIntrospectionTest.java

```diff
@@ -0,0 +1,139 @@
+package com.inngest.springbootdemo;
+
+import com.inngest.*;
+import com.inngest.signingkey.BearerTokenKt;
+import com.inngest.signingkey.SignatureVerificationKt;
+import com.inngest.springboot.InngestConfiguration;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfSystemProperty;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.web.servlet.MockMvc;
+import uk.org.webcompere.systemstubs.environment.EnvironmentVariables;
+import uk.org.webcompere.systemstubs.jupiter.SystemStub;
+import uk.org.webcompere.systemstubs.jupiter.SystemStubsExtension;
+
+import java.util.HashMap;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+class ProductionConfiguration extends InngestConfiguration {
+
+    public static final String INNGEST_APP_ID = "spring_test_prod_demo";
+
+    @Override
+    protected HashMap<String, InngestFunction> functions() {
+        return new HashMap<>();
+    }
+
+    @Override
+    protected Inngest inngestClient() {
+        return new Inngest(INNGEST_APP_ID);
+    }
+
+    @Override
+    protected ServeConfig serve(Inngest client) {
+        return new ServeConfig(client);
+    }
+
+    @Bean
+    protected CommHandler commHandler(@Autowired Inngest inngestClient) {
+        ServeConfig serveConfig = new ServeConfig(inngestClient);
+        return new CommHandler(functions(), inngestClient, serveConfig, SupportedFrameworkName.SpringBoot);
+    }
+}
+
+@ExtendWith(SystemStubsExtension.class)
+public class CloudModeIntrospectionTest {
+
+    private static final String productionSigningKey = "signkey-prod-2a89e554826a40672684e75eee6e34909b45aa4fd04fff5ff49bbe28c24ef424";
```

Comment:
i generated a signing key and event key on the platform and deleted them. In the end I didn't even use the event key because i wanted to use the same value/hash combination that the Python sdk is using in: https://github.com/inngest/inngest-py/blob/6dcae0b4726f8edaa3d45015fdca9b21be7486b6/tests/base.py#L199
I'll remove this one as well.

Reply:
Ok cool, using the same as Python sdk also sounds good to me